ocfweb/account/register.py

```diff
@@ -108,6 +113,38 @@ def request_account(request):
     )
 
 
+def recommend(request):
+    real_name = request.GET.get('real_name', '')
+    if real_name == '':
+        return JsonResponse({'recommendations': []})  # Return empty if no real name
+
+    recommendations = recommender.recommend(real_name, 3)
+    return JsonResponse(
+        {
+            'recommendations': recommendations,
+        },
+    )
+
+
+def validate(request):
+    real_name = request.GET.get('real_name', '')
+    username = request.GET.get('username', '')
+    try:
+        validate_username(username, real_name)
+        return JsonResponse({
+            'is_valid': True,
+            'msg': 'Username is available.',
+        })
+    except (ValidationError, ValidationWarning) as e:
+        msg = str(e)
+        if msg[-1] != '.':
+            msg += '.'
```

Comment:
Nice.

Reply:
Modifying the exception messages sounds like a better idea. Made a separate ocflib PR for that.